### PR TITLE
Remaining JS linting TODO

### LINKED ISSUE #28455: Remaining JS linting TODO
* [x] simplify one-line multi-line imports when we have one thing we import.
* [ ] take care of the max-depth warning in button.js
* [ ] enable `no-mixed-operators` rule

### PR DESCRIPTION
**Pull request by mcmxcdev**
Since there was no activity but the "help wanted" label assigned to the issue, I took over, hope that is fine.

Why are the `js/dist`, `site/static` and `dist/*` folders not in `.gitignore`? They showed up in my changed files when running `yarn test`.

Fixes #28455

### CODE REVIEW

**Comment by XhmikosR** [comment]
Thanks for the patch! LGTM from a quick look. I'll wait for @Johann-S's approval too.

>Why are the `js/dist`, `site/static` and `dist/*` folders not in `.gitignore`? They showed up in my changed files when running `yarn test`.

Because we have them in git on purpose since the beginning. You could just run `npm run js-lint`.

**Review by Johann-S — APPROVED** [review]
LGTM 👍 Thanks @mhatvan 👍

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**
